timesketch/frontend-ng/src/components/Explore/EventList.vue

```diff
@@ -145,6 +156,10 @@ limitations under the License.
                 </v-card>
               </v-dialog>
 
+              <v-btn icon @click="exportSearchResult()">
+                <v-icon>mdi-download</v-icon>
+              </v-btn>
```

Comment:
```suggestion
              <v-tooltip top open-delay="500">
                <template v-slot:activator="{ on }">
                  <v-btn v-on="on" icon @click="exportSearchResult()">
                <v-icon>mdi-download</v-icon>
                  </v-btn>
                </template>
                <span>Download current view as csv</span>
              </v-tooltip>
```

UX nit: Add a tooltip to the icon.